contributing_to_docs/doc_guidelines.adoc

```diff
@@ -248,21 +240,20 @@ $ lorem.sh
 
 Without the line spaces the content is likely to be not parsed correctly.
 
-=== Inline Code or Commands
-Do NOT show full commands or command syntax inline within a sentence. See <<Command syntax and examples>> for information on how to show commands and command syntax.
+=== Command syntax and examples
+Code blocks are used to show command usage, syntax, and examples of command screen outputs or configuration files. Code blocks can be used in a few different ways:
 
-Only use case for inline commands would be general commands and operations, without replaceables and command options. In this case an inline command is marked up using the back ticks:
+1. You will use a code block when showing the initial usage of a particular command, or the full command syntax with the required options and parameters.
 
-....
-Use the `GET` operation to do x.
-....
+2. Code blocks are also used to show real examples of commands. 
 
-This renders as:
+Follow these general guidelines when using code blocks:
 
-Use the `GET` operation to do x.
+* Always show command syntax or usage with any parameters or values as <replaceables>
 
-=== Command syntax and examples
-The main distinction between showing command syntax and example is that a command syntax should just show customers how to use the command without real values. An example on the other hand should show the command with actual values with an example output of that command, where applicable.
+* If the command or command syntax warrants a full example, wrap it inside an example block, and use actual example values. Do NOT use <replaceables> in command example outputs.
+
+* Try to use callouts to provide information on what the output represents when required
```

Comment:
@CowboysFan I'd suggest this "Command Syntax and Examples" section above be condensed a little. We've got two lists (one using numbers, and the other dot points) that are for the same purpose. The list items are even quite similar. I'd suggest that the first (numbered) list is redundant, as the info there is either in the little intro para, or the dot point list.